templates/allauth/account/signup.html

```diff
@@ -0,0 +1,21 @@
+{% extends "account/base.html" %}
+
+{% load i18n %}
```

Comment:
This reminds me...  I should probably open an issue for `i18n` support on gitcoin.

Reply:
I need to start using it everywhere